requirements.txt

```
@@ -7,3 +7,6 @@ git-changelog
 git+https://github.com/dimrozakis/mkpdfs-mkdocs-plugin.git@master
 pygments
 weasyprint<53
+matplotlib==3.3
+pybids
+nilearn
```

```suggestion
nilearn==0.9
```
In the pre-registration, we precised that we will use nilearn version 0.9.

---

I read that, but I feel like a lot of nilearn's processes really are outdated in version 0.9. Most of the internal methods of the extraction and denoising of timeseries will have to be re-made by hand to have correct results. I'll make it work with nilearn 0.10 for now and I'll add compatilibity patches later if need be.